code/presets/react-webpack/src/framework-preset-react-docs.ts

```diff
@@ -60,8 +62,9 @@ export const webpackFinal: StorybookConfig['webpackFinal'] = async (
           ),
           options: {
             babelOptions,
+            debug,
           },
-          exclude: /node_modules/,
+          exclude: /(\.stories\.(js|jsx|ts|tsx))|(node_modules)/,
```

Comment:
I think we officially support only `.stories.*` and `.story.*`

Reply:
Via the `stories` option in main.ts, the user might even define more options. Supporting the default and recommended naming convention is in my point of view enough.

- I don't feel e.g. safe to add `stories.*`. This might exclude all files with `stories` in the name. e.g. `lovestories.tsx`
- Same for `story.*`

So arguable I could add `.story.` as the only addition here. But again. I think people should follow the recommendation and its just about performance optimizations. The former `babel-plugin-react-docgen` plugin processed everything. So it is definitely an improvement for 90% of Storybook users.

Reply:
Added `.story.*`